content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,471 @@
+are activated and preconfigured with the correct tax grids. These are set up automatically to ensure
+the correct booking of accounting entries and display of the tax report.
+
+For **vendor bills**, **three** types of configurations are technically identified by a code called
+:guilabel:`Tipo Documento`:
+
+- `TD17` - Buying services from **EU** and **non-EU** countries;
+- `TD18` - Buying **goods** from **EU**;
+- `TD19` - Buying **goods** from a **foreign** vendor, but the **goods** are already in **Italy**
+  in a **VAT deposit**.
+
+.. tabs::
+
+   .. tab:: `TD17`
+
+      Buying **services** from **EU** and **non-EU** countries:
+
+      The foreign *seller* invoices a service with a **VAT-excluded** price, as it is not
+      taxable in Italy. The VAT is paid by the *buyer* in Italy;
+
+      - Within EU: the *buyer* integrates the invoice received with the **VAT information**
+        due in Italy (i.e., **vendor bill tax integration**);
+      - Non-EU: the *buyer* sends themselves an invoice (i.e., **self-billing**).
+
+      Odoo exports a transaction as `TD17` if the following conditions are met:
+
+      - Is a vendor bill;
+      - At least one tax on the invoice lines targets the tax grids :ref:`VJ <italy/grids>`;
+      - All invoice lines either have :guilabel:`Services` as **products**, or a tax with the
+        :guilabel:`Services` as **tax scope**.
+
+  .. tab:: `TD18`
+
+     Buying **goods** from **EU**:
+
+     Invoices issued within the EU follow a **standard format**, therefore only an integration of
+     the existing invoice is required.
+
+     Odoo exports a transaction as `TD18` if the following conditions are met:
+
+     - Is a vendor bill;
+     - At least one tax on the invoice lines targets the tax grids :ref:`VJ <italy/grids>`;
+     - All invoice lines either have :guilabel:`Consumable` as **products**, or a tax with the
+       :guilabel:`Goods` as **tax scope**.
+
+  .. tab:: `TD19`
+
+     Buying **goods** from a **foreign** vendor, but the **goods** are already in **Italy** in a
+     **VAT deposit**:
+
+     - From EU: the *buyer* integrates the invoice received with the **VAT information** due in
+       Italy (i.e., **vendor bill tax integration**);
+     - Non-EU: the *buyer* sends an invoice to *themselves* (i.e., **self-billing**).
+
+     Odoo exports a move as a `TD19` if the following conditions are met:
+
+     - Is a vendor bill;
+     - At least one tax on the invoice lines targets the tax grid :ref:`VJ3 <italy/grids>`;
+     - All invoice lines either have :guilabel:`Consumable` products, or a tax with
+       :guilabel:`Goods` as tax scope.
+
+.. _italy/internal-reverse:
+
+Internal reverse charge
+-----------------------
+
+.. warning::
+   Odoo currently does not support **internal reverse charge** processes.
+
+In some cases, vendor bills have the VAT duty **reverse charged** on the *buyer* for **national
```

Comment:
@chiaraprattico I don't know why we put an entire explanation of something we do not support...

Reply:
So I can erase all the paragraph from 371 to 381?

Reply:
yep (and maybe in L 369 put internal (=domestic) reverse charge @lordkrandel

Reply:
The idea is that probably we'll recycle the precious information for a more technical internal document, Chiara ;) Then, when we implement things, it will be restored here.